### ISSUE TITLE
Accessing Existing/Non-Existing variables in onEnd not working

### ISSUE THREAD

**Issue by naqvis** [flomesh-io/pipy#113]
## Pipy version

Using the latest commit 

```shell
Version     : 0.50.0-18
Commit      : b80b1bc464f95b3ad6c2f2b38c2e7c0acc065877
Commit Date : Thu, 25 Aug 2022 11:47:51 +0800
Host        : Darwin-21.6.0 arm64
OpenSSL     : OpenSSL 1.1.1q  5 Jul 2022
Builtin GUI : Yes
Samples     : Yes
```

## Bug Report

* Accessing Global/Context variables in `onEnd` is not working.
* Accessing any invalid or non-existing variable in `onEnd` doesn't generate any error/warning

### Problem 1

For example trying to access any valid variable doesn't yield any outcome.

```js
 .onEnd(
  () => console.log('print some global variable', __inbound.remoteAddress)
)
```

Won't print anything, but logging any string literal works.

### Problem 2

For example trying to access any invalid or non-existing variable doesn't yield any error/warning.

```js
 .onEnd(
  () => console.log('print some non existing variable', _not_existing_variable)
)
```
No error/warning is generated.

**Comment by pajama-coder** [comment]
This bug is solved in a9474a551866a89388420f99a144316301236309.
Please verify it @naqvis
Thanks.

**Comment by naqvis** [comment]
> This bug is solved in [a9474a5](https://github.com/flomesh-io/pipy/commit/a9474a551866a89388420f99a144316301236309). Please verify it @naqvis Thanks.

Thank you @pajama-coder. I verified the latest commit and its fixed. So closing this issue.
